libensemble/libE_manager.py

```diff
@@ -435,7 +435,10 @@ def _alloc_work(self, H, persis_info):
         fields before the alloc_f call and ensures they weren't modified
         """
         if self.safe_mode:
-            saveH = copy.deepcopy(H[protected_libE_fields])
+            if 'repack_fields' in globals():
+                saveH = copy.deepcopy(repack_fields(H[protected_libE_fields]))
```

Comment:
Do you need to do deepcopy and repack_fields in same line? Surely repack gives you deep copy.